crypto/src/main/java/org/cryptimeleon/incentive/crypto/BilinearGroupChoice.java

```diff
@@ -0,0 +1,10 @@
+package org.cryptimeleon.incentive.crypto;
```

Comment:
why is that entirely new? refactoring?

Reply:
Yes, It used to be in the setup class, but it is better practise to have enums in separate files